app/presenters/flow_presenter.rb

```diff
@@ -54,6 +54,14 @@ def change_answer_link(question, forwarding_responses)
     end
   end
 
+  def change_link_track_label(question, accepted_response)
+    if question.node_name == :vaccination_status
```

Comment:
It feels a bit odd to have a reference to a flow specific question `vaccination_status` in the generic Flow Presenter.

Is it possible to delegate this method the "Flow" class? Or does "Flow" not have access to the `question` object? Or maybe QuestionPresenter?

I need a little time to think about this.

Reply:
I can give it a try :) The reason I went with flow_presenter was that it already had `change_answer_link` and the change answer section is on all of the flows. Agree that having the `vaccine_status` specific stuff in there is a bit weird but I thought it would be ok as it also displayed the label for the other questions... however if I can delegate it and then make use of inheritance then than would be better so i'll see if I can... thanks!

Reply:
That approach did work but testing it was really difficult and it didn't feel right having an overriding method in the `check_travel_during_coronavirus_flow` because it wasn't in keeping with our usual flow test style. I came up with a new solution which I think works better, where the method lives on the `question_presenter`. Let me know what you think :) @leenagupte